WordPress/Classes/ViewRelated/NUX/SiteCreationDomainSearchTableViewCell.xib

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The border at the top of the search bar doesn't go all the way across to the left:
<img width="536" alt="search_bar" src="https://user-images.githubusercontent.com/1816888/34179588-d7eb648c-e4c8-11e7-9a68-694362f9e9f0.png">